test/studies/lcals/LongDouble.chpl

```diff
@@ -0,0 +1,39 @@
+/* A module to provide a "long double" type to Chapel that matches the
```

Comment:
I understand the need for it, but it's a shame to have to have this helper module. Using GMP seems like a bad idea because that would add an odd dependency in order to run lcals. I guess I'm wondering if we could make this more of a standard module /first class type instead of needing a helper module for it (presumably other users could have use for a more precision floating point type too.)

Anyways, I think this is fine for now, just thinking out loud.